recipe/meta.yaml

```
@@ -11,16 +11,16 @@ source:
 
 build:
   noarch: python
-  number: 2
+  number: 3
   script: python -m pip install --no-deps --ignore-installed .
```

Let's take this opportunity and update this line to:

```suggestion
script: {{ PYTHON }} -m pip install . -vv
```

---

Can you also remove the line `fn: termcolor-{{ version }}.tar.gz` from above?

---

sure, thanks!